dev/NavigationView/NavigationViewItem.cpp

```diff
@@ -867,12 +966,32 @@ void NavigationViewItem::ProcessPointerCanceled(const winrt::PointerRoutedEventA
     }
 
     m_isPressed = false;
-    m_isPointerOver = false;
+    // m_isPointerOver should be true before this event so this doesn't need to be set to true in the else block...
+    // What this flag tracks is complicated because of the NavigationView sub items and the m_capturedPointers that are being tracked..
+    // We do this check because PointerCaptureLost can sometimes take the place of PointerReleased events.
+    // In these cases we need to test if the pointer is over the item to maintain the proper state.
```

Comment:
Unrelated to this PR: these flags keep piling up, how should this continue in the future? state becomes quite hard to track in NavigationView @ranjeshj FYI @ojhad FYI

Reply:
Agreed, NavViews complexity continues to grow.